Lib/test/test_os.py

```diff
@@ -3524,6 +3527,89 @@ def test_memfd_create(self):
         self.assertFalse(os.get_inheritable(fd2))
 
 
+@unittest.skipUnless(hasattr(os, 'eventfd'), 'requires os.eventfd')
+@support.requires_linux_version(2, 6, 30)
+class EventfdTests(unittest.TestCase):
+    def test_eventfd_initval(self):
+        def pack(value):
+            """Pack as native uint64_t
+            """
+            return struct.pack("@Q", value)
+        size = 8  # read/write 8 bytes
```

Comment:
I think this should be `struct.calcsize("L")`

Reply:
According to the example in the man page, it's an uint64_t in native format, so ``@Q`` is correct.